test-data/unit/check-custom-plugin.test

```diff
@@ -143,19 +143,49 @@ tmp/mypy.ini:2: error: Return value of "plugin" must be a subclass of "mypy.plug
 [case testAttributeTypeHookPlugin]
 # flags: --config-file tmp/mypy.ini
 from typing import Callable
-from m import Signal
+from m import Signal, DerivedSignal
 s: Signal[Callable[[int], None]] = Signal()
 s(1)
 s('') # E: Argument 1 has incompatible type "str"; expected "int"
+
+ds: DerivedSignal[Callable[[int], None]] = DerivedSignal()
+ds('') # E: Argument 1 has incompatible type "str"; expected "int"
 [file m.py]
 from typing import TypeVar, Generic, Callable
 T = TypeVar('T', bound=Callable[..., None])
 class Signal(Generic[T]):
     __call__: Callable[..., None]  # This type is replaced by the plugin
+
+class DerivedSignal(Signal, Generic[T]): ...
```

Comment:
I think this should be `Signal[T]`, this will fix the test.

Reply:
Hm, it looks like variables (unlike methods) are passed with non-mapped instance type, so your fix is right. Anyway, it i better to change this to `Signal[T]` (currently it is equivalent to `Signal[Any]`).